pkg/reconciler/v1alpha1/route/table_test.go

```diff
@@ -474,6 +483,9 @@ func TestReconcile(t *testing.T) {
 			),
 			resources.MakeK8sService(simpleRunLatest("default", "new-latest-created", "config", nil)),
 		},
+		WantPatches: []clientgotesting.PatchActionImpl{
+			patchLastPinned("default", "config-00001"),
```

Comment:
Feels like we could avoid most of these diffs by making the default `lastPinned` value close enough to the clock to elide them.  Then we'd only need to adjust the `lastPinned` value on the cases we are intentionally testing?

Reply:
Good idea! Done.